ua-snap/geospatial-vector-veracity
# Add NCR places for the EPA-Justice project

This PR adds the "missing" communities identified in the EPA-Justice project work. These communities all have corresponding census designated places (CDPs) that we are hoping to match up 1:1 when a user queries NCR for a community name.

All of these additional communities (or more specifically, their coordinates) were vetted to make sure that NCR returns at least some data when queried. Two of the desired locations (Adak and Eareckson Station) did not return any NCR data, and were therefore removed from the list.

Alaskan native language place names were included only if the community was listed in this table: https://www.uaf.edu/anla/collections/map/names/ . Otherwise these are left NA.

The distance to ocean attribute of the table was also recalculated for consistency, though apparently this field has not yet been used for anything in particular. 

From what I can tell, none of the original IDs (e.g., "AK###") were affected by the additions due to the way the `utilities.add_point_location.py` was written. I had to update some of the functions in this module to work with pandas > 2.0, but all of those changes are in a copy that lives in the `epa-justice` project repo and there is no need to update the scripts in this GVV repo.

**TO TEST:**

Review all the table processing in [this notebook](https://github.com/ua-snap/epa-justice/blob/main/update_NCR_points.ipynb) in the `epa-justice` project repo. Most of the work here (identifying coordinates for communities, checking returns in NCR) was a manual process. But this work would definitely benefit from a quick sanity check on the table-wrangling code and a check for typos / wacky values in the modified lines of `alaska_point_locations.csv`.

Also, ponder the question: is there anything else to be done here? Does adding communities to this table have any downstream effects (in NCR or elsewhere?).
